Doc/library/sqlite3.rst

```diff
@@ -294,7 +295,7 @@ Module functions
        controlling whether and how transactions are implicitly opened.
        Can be ``"DEFERRED"`` (default), ``"EXCLUSIVE"`` or ``"IMMEDIATE"``;
        or ``None`` to disable opening transactions implicitly.
-       See :ref:`sqlite3-controlling-transactions` for more.
+       See :ref:`sqlite3-transaction-control-isolation-level` for more.
```

Comment:
> This still needs at least a brief mention that autocommit is recommended [...]

+1

> Furthermore, doesn't this now have no effect when autocommit is not set to the legacy default? If so, that should be mentioned [...]

+1

> Furthermore, if that is the case, is there a reason to allow passing this at all if users have opted into the new behavior, since it will silently be ignored if so?

Well, it won't have any effect, since those lines of code dealing with `isolation_level` will never run if `autocommit` is True or False. So, "ignored" is not quite right.

> Why not make it a warning or a an error, rather than letting what is likely an mistake pass silently?

That _may_ be a better alternative. I'm not sure deciding on this should block this PR, though. We can add a warning later in the alpha phase.

Reply:
Actually, I believe we're better off with keeping this section short, refer to the `isolation_level` _attribute_, and expand that section. This means less repetition.

Reply:
> Well, it won't have any effect, since those lines of code dealing with `isolation_level` will never run if `autocommit` is True or False. So, "ignored" is not quite right.

In colloquial English usage, they are effectively synonymous, but technically true (the best kind), I guess, if you want to be pedantic about it :stuck_out_tongue_winking_eye: though 

> That may be a better alternative. I'm not sure deciding on this should block this PR, though. We can add a warning later in the alpha phase

:+1: 

> Actually, I believe we're better off with keeping this section short, refer to the isolation_level attribute, and expand that section. This means less repetition.

:+1:

Reply:
FTR, I won't touch the `isolation_level` param docs more in this PR. We can make it more succinct in a separate PR.